nlp/2021_06_18_data_augmentation/totw_nlp_dat_aug.ipynb

```diff
@@ -0,0 +1,2949 @@
+              "      <td>0.677472</td>\n",
+              "    </tr>\n",
+              "    <tr>\n",
+              "      <td>175</td>\n",
+              "      <td>0.004700</td>\n",
+              "      <td>1.391002</td>\n",
+              "      <td>0.682334</td>\n",
+              "    </tr>\n",
+              "    <tr>\n",
+              "      <td>200</td>\n",
+              "      <td>0.003400</td>\n",
+              "      <td>1.441377</td>\n",
+              "      <td>0.683955</td>\n",
+              "    </tr>\n",
+              "    <tr>\n",
+              "      <td>225</td>\n",
+              "      <td>0.002800</td>\n",
+              "      <td>1.485793</td>\n",
+              "      <td>0.675851</td>\n",
+              "    </tr>\n",
+              "    <tr>\n",
+              "      <td>250</td>\n",
+              "      <td>0.002500</td>\n",
+              "      <td>1.515399</td>\n",
+              "      <td>0.674230</td>\n",
+              "    </tr>\n",
+              "    <tr>\n",
+              "      <td>275</td>\n",
+              "      <td>0.002200</td>\n",
+              "      <td>1.527388</td>\n",
+              "      <td>0.682334</td>\n",
+              "    </tr>\n",
+              "    <tr>\n",
+              "      <td>300</td>\n",
+              "      <td>0.002100</td>\n",
+              "      <td>1.532375</td>\n",
+              "      <td>0.680713</td>\n",
+              "    </tr>\n",
+              "    <tr>\n",
+              "      <td>325</td>\n",
+              "      <td>0.001900</td>\n",
+              "      <td>1.541955</td>\n",
+              "      <td>0.680713</td>\n",
+              "    </tr>\n",
+              "    <tr>\n",
+              "      <td>350</td>\n",
+              "      <td>0.001800</td>\n",
+              "      <td>1.546143</td>\n",
+              "      <td>0.680713</td>\n",
+              "    </tr>\n",
+              "  </tbody>\n",
+              "</table><p>"
+            ],
+            "text/plain": [
+              "<IPython.core.display.HTML object>"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          }
+        },
+        {
+          "output_type": "display_data",
+          "data": {
+            "text/html": [
+              "\n",
+              "    <div>\n",
+              "      \n",
+              "      <progress value='78' max='78' style='width:300px; height:20px; vertical-align: middle;'></progress>\n",
+              "      [78/78 00:02]\n",
+              "    </div>\n",
+              "    "
+            ],
+            "text/plain": [
+              "<IPython.core.display.HTML object>"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          }
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "cxt1VqkmU84t"
+      },
+      "source": [
+        "### Model back-translated & contextual word embedding insertions"
```

Comment:
I would still add a section after this one where all of the augmented sets are combined

Reply:
Since we saw that this performs consistently very well